src/test/java/utils/TestDataHelper.java

```diff
@@ -0,0 +1,49 @@
+package utils;
+
+import enums.DistanceType;
+import enums.Feeling;
+import enums.PerceivedEffort;
+import enums.PaceType;
+import models.Bike;
+import models.PrintWorkouts;
+import models.Workout;
+
+public class TestDataHelper {
```

Comment:
Try to use some random data generators for test data like java-faker: https://github.com/DiUS/java-faker